pic-sure-auth-ui/src/main/webapp/admin/styles.css

```diff
@@ -219,35 +219,35 @@
     background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
 }
 
-.table-users {
+.table-picsure {
```

Comment:
we may need to update User mngt, cause css change like this will break the layout on that page.

Reply:
right, I'll update the user mngt as well